clang/include/clang/AST/DeclBase.h

```diff
@@ -1819,10 +1812,10 @@ class DeclContext {
     SourceLocation AtStart;
   };
 
-  /// Number of non-inherited bits in ObjCContainerDeclBitfields.
+  /// Number of inherited and non-inherited bits in ObjCContainerDeclBitfields.
   /// Note that here we rely on the fact that SourceLocation is 32 bits
   /// wide. We check this with the static_assert in the ctor of DeclContext.
-  enum { NumObjCContainerDeclBits = 64 - NumDeclContextBits };
+  enum { NumObjCContainerDeclBits = 64 };
```

Comment:
As we discussed in private, `sizeof` yields size in bytes, so there is a mismatch in units. But it doesn't work even after fixing this, because `ObjCContainerDeclBitfields` is a rare Bitfields type that relies on implicit padding bits, and has a non-bit-field member for that. So I'm not applying this suggestion.